src/pages/PersonalProfile/PersonalProfile.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import './PersonalProfile.css';
+import ProfileCard from './ProfileCard';
+import ProfileContent from './ProfileContent';
+import USER_DATA from './USER_DATA';
+
+const Profile = styled.div`
+    display: flex;
+    width: 80vw;
```

Comment:
This ` width: 80vw;` css property, not good with responsiveness
![width-rule](https://user-images.githubusercontent.com/38657595/61593765-77ad7a00-abec-11e9-94b9-d17652cbf065.jpg)